packages/common/utils/src/range.ko.md

```diff
@@ -9,7 +9,7 @@ range(
   // 시작할 숫자
   start: number,
   // 끝날 숫자 (주의: `end` 는 포함되지 않습니다)
-  end: number,
+  end?: number,
```

Comment:
It would be nice to documentate what the default value of `end` is.

Reply:
I think that's a good idea!
The default value of the `end` value is not specified on the code and if implicitly undefined, it follows the value of start, so should I correct it as follows?

[fix: @toss/utils declare default value for end parameter for UX](https://github.com/toss/slash/pull/116/commits/f2e8bd69ae1fe86758344ce487723487226ee206)
[f2e8bd6](https://github.com/toss/slash/pull/116/commits/f2e8bd69ae1fe86758344ce487723487226ee206)

Reply:
Yes! Thanks!